css/section.css

```diff
@@ -0,0 +1,15 @@
+.section {
+  padding: 94px 0;
+}
+
+.section__title {
+  margin: 0 0 50px;
+}
+
+.section--grey {
+  background-color: var(--section-color);
+}
+
+.section--pd-btm-0 {
+  padding-bottom: 0;
+}
```

Comment:
you already have a class for section without bottom margin - re-use it in the project